Cargo.toml

```diff
@@ -20,6 +20,7 @@ colorized = "1"
 derive_builder = "0.13"
 directories = "5"
 env_logger = "0.11"
+format_serde_error = { git = "https://github.com/duyet/format_serde_error" }
```

Comment:
I think we should be using a published crate instead, unless there's a change you're wanting to use that isn't published yet

Reply:
The core crate is unmaintained and abandoned. So we either find a new crate or fork it and maintain it ourselves

Reply:
Oh really? Well that sucks. What changes in this git version are there that we're using and doesn't exist in the published one?

Reply:
They just bump serde yaml to 0.9

Reply:
Ah ok, yeah that's fairly important lol